### PR TITLE
Option to include request body in log util

### PR DESCRIPTION
**Pull request by sshniro**
Partially fixes: #1512

I was looking through how to get the response body from the ngx variable, but unfortunately, I could not find any reference.

### CODE REVIEW

**Comment by sshniro** [comment]
It seems all the logger should be updated at the same time, I'll update the loggers in this PR.

File: apisix/utils/log-util.lua [hunk 1]
```diff
@@ -56,6 +56,12 @@ local function get_full_log(ngx)
         start_time = ngx.req.start_time() * 1000,
         latency = (ngx.now() - ngx.req.start_time()) * 1000
     }
+
+    if conf.include_req_body then
+        log.request.body = ngx.req.get_body_data()
```
Comment (spacewander) [inline apisix/utils/log-util.lua:61]: Should we distinguish `without body` (like `GET` request) from `body too big` (like `get_body_data()` returns `nil` but `get_body_file()` doesn't)?

Comment (moonming) [reply apisix/utils/log-util.lua:61]: yes, I think we shoud do.

Comment (sshniro) [reply apisix/utils/log-util.lua:61]: agreed, I'll make the changes accordingly.

**Comment by membphis** [comment]
@sshniro please take a look at the output of Travis:

https://travis-ci.org/github/apache/incubator-apisix/jobs/683698207#L836

**Comment by sshniro** [comment]
Sure @membphis , I'll fix the issues during this weekend.

**Comment by sshniro** [comment]
@spacewander @moonming should be fixed by now. Please have a look.

**Comment by membphis** [comment]
@sshniro please add test case for your new code

**Comment by sshniro** [comment]
@membphis testing the log utils is a bit complicated as we need an ngx variable. I thought to test the log util via the HTTP logger.

**Comment by membphis** [comment]
ok, got it. we can merge it first

**Comment by membphis** [comment]
@sshniro many thx, merged already

**Pull request merged**